.github/workflows/perf_test.yaml

```
@@ -0,0 +1,31 @@
+name: V2 Performance Test
+
+on:
+  schedule:
+    - cron: "0 18 * * *" # currently run daily on 18:00 UTC; [TODO]: run weekly
```

Generally there would be no changes on the weekends. and currently v1.x daily test are scheduled on UTC 19 so I think the CI resources could be occupied too long  for running daily testing.

```suggestion
    - cron: "0 16 * * 1-5" # currently run daily on 16:00 UTC; [TODO]: run weekly
```